test/e2e/components/vl-map-layer.js

```diff
@@ -13,11 +13,19 @@ class VlMapLayer extends VlElement {
     return this.driver.executeScript(`return arguments[0].layer.getSource().getFeatures();`, this);
   }
 
+  async getAantalFeatures() {
```

Comment:
Engels ipv nederlands?

Reply:
ja tom wil engels in webcomponenten denk ik ja